drivers/spi/spi_ll_stm32.c

```diff
@@ -272,6 +272,60 @@ static int spi_dma_move_buffers(const struct device *dev, size_t len)
 /* Value to shift out when no application data needs transmitting. */
 #define SPI_STM32_TX_NOP 0x00
 
+static void spi_stm32_send_next_frame(SPI_TypeDef *spi,
+		struct spi_stm32_data *data)
+{
+	const uint8_t frame_size = SPI_WORD_SIZE_GET(data->ctx.config->operation);
```

Comment:
It would be cleaner and easier to retrieve. However, the same information would be replicated in two places, with the inconsistency problems that could carry. I don't have a strong opinion on this so I'm happy to add a new attribute to the struct.

Reply:
More pointer dereference might increase the risk of cache miss. And since in this case you are only reaching for a small info, so better storing it in data right away

Reply:
Agreed.

Reply:
I have taken a look at this. There are other places in which `ctx.config` is accessed; to be consistent, all these references should be modified. This is out of this PR's scope as I conceived it.